autoPyTorch/evaluation/time_series_forecasting_train_evaluator.py

```diff
@@ -146,10 +146,7 @@ def __init__(self, backend: Backend, queue: Queue,
         self.num_sequences = self.datamanager.num_sequences
         self.num_targets = self.datamanager.num_targets
         self.seq_length_min = np.min(self.num_sequences)
-        seasonality = SEASONALITY_MAP.get(self.datamanager.freq, 1)
-        if isinstance(seasonality, list):
-            seasonality = min(seasonality)  # Use to calculate MASE
-        self.seasonality = int(seasonality)  # type: ignore[call-overload]
+        self.seasonality = self.datamanager.seasonality
```

Comment:
Hey,

In the `AbstractEvaluator` class, we use a `_init_datamanager_info` (see [here](https://github.com/automl/Auto-PyTorch/blob/17cdc09f4b748174f5434a0cb5572b373cced5e4/autoPyTorch/evaluation/abstract_evaluator.py#L551)). At the end of this method we delete the datamaneger object to reduce the memory usage. Could you also update the time series forecasting evaluator, maybe overriding the parent class function?

Reply:
you can also do it as a separate PR

Reply:
okay sure.